projects/step-frontend/src/lib/modules/timeseries/performance-view/performance-view.component.ts

```diff
@@ -161,14 +161,18 @@ export class PerformanceViewComponent implements OnInit, OnDestroy {
       .subscribe((keywords) => {
         this.keywords = keywords;
       });
-    this.executionContext
+    this.executionContext.timeSelectionState
       .onActiveSelectionChange()
       .pipe(takeUntil(this.subscriptionsTerminator$))
       .subscribe((newRange) => {
         this.timeSelection = newRange;
-        this.updateTable();
+        this.updateTable().subscribe();
```

Comment:
Usually, when subscribe invocation appears inside another subscribe - it's a sign that observable chain should be reorganized. For example this one can be implemented as:
```
this.executionContext.timeSelectionState
     .onActiveSelectionChange()
     .pipe(
         tap((newRange) => this.timeSelection = newRange),
         switchMap(() => this.update()),
         takeUntil(this.subscriptionsTerminator$)
     )
     .subscribe();
```